cmd/rollkit/commands/intercept.go

```diff
@@ -65,7 +65,7 @@ readTOML:
 	return true, runEntrypoint(&rollkitConfig, flags)
 }
 
-func buildEntrypoint(rootDir, entrypointSourceFile string, forceRebuild bool) (string, error) {
+func entrypointBinaryFilePath(rootDir, entrypointSourceFile string, forceRebuild bool) (string, error) {
```

Comment:
The function name was fine, because the function is building the entry point. 

My comment was on the variable names.